pkg/vpc/vpc.go

```diff
@@ -65,6 +76,31 @@ func SetSubnets(vpc *api.ClusterVPC, availabilityZones []string) error {
 	return nil
 }
 
+func SplitInto16(parent *net.IPNet) ([]*net.IPNet, error) {
```

Comment:
Ooh can we get rid of the `kops` dependency here and just include our own `SplitInto8`?

Reply:
will do!